types/firefox-webext-browser/firefox-webext-browser-tests.ts

```diff
@@ -0,0 +1,2 @@
+// No tests yet
+true;
```

Comment:
While I was experimenting a bit with this (I had not previous experience with typescript or DefinitelyTyped until now), I started to put something in this file, it looks a pretty useful way to test the generated types (especially if you want to be sure that the types are still generated correctly while tweaking the generator script), here is some example tests that I added into it:

```
browser.nonexistentNS.unknownMethod(); // $ExpectError

// Various API types tests
browser.runtime.getManifest(); // $ExpectType ManifestType
```